src/utils/fecha.js

```diff
@@ -234,8 +237,11 @@ parseFlags.mm = parseFlags.m;
 parseFlags.hh = parseFlags.H = parseFlags.HH = parseFlags.h;
 parseFlags.ss = parseFlags.s;
 parseFlags.A = parseFlags.a;
+parseFlags.XXX = parseFlags.XX = parseFlags.X;
 
 export const format = (dateObj, mask, locale) => {
+  mask = (isArray(mask) && mask) || (isString(mask) && mask) || 'YYYY-MM-DD';
```

Comment:
https://github.com/nathanreyes/v-calendar/blob/a806fb8334cdb1c6c61ef0ddacb6f2db9f298046/src/utils/fecha.js#L254

`mask.replace` fails when mask is an array. `TypeError: mask.replace is not a function`

See the "Popover Mode" example on the homepage